Bottom: 3b074bf5245e6007114fb305b02f837b1c1c6ade
Top:    100e266652f60367706725e156902c99f6fcd27b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-20 00:06:01 +0000

Refresh of hid-lpr-initiialise-common-dra

---

diff --git a/src/hid/lpr/lpr.c b/src/hid/lpr/lpr.c
index 027394d..a03b19c 100644
--- a/src/hid/lpr/lpr.c
+++ b/src/hid/lpr/lpr.c
@@ -16,7 +16,6 @@
 #include "../hidint.h"
 #include "../ps/ps.h"
 #include "hid/common/hidnogui.h"
-#include "hid/common/draw_helpers.h"
 #include "hid/common/hidinit.h"
 
 #ifdef HAVE_LIBDMALLOC
@@ -121,18 +120,14 @@ lpr_calibrate (double xval, double yval)
 }
 
 static HID lpr_hid;
-static HID_DRAW_API lpr_graphics;
 
 void
 hid_lpr_init ()
 {
   memset (&lpr_hid, 0, sizeof (HID));
-  memset (&lpr_graphics, 0, sizeof (HID_DRAW_API));
 
   common_nogui_init (&lpr_hid);
   ps_ps_init (&lpr_hid);
-  common_draw_helpers_init (&lpr_hid);
-  ps_ps_graphics_init (&lpr_graphics);
 
   lpr_hid.struct_size         = sizeof (HID);
   lpr_hid.name                = "lpr";
@@ -145,7 +140,5 @@ hid_lpr_init ()
   lpr_hid.parse_arguments     = lpr_parse_arguments;
   lpr_hid.calibrate           = lpr_calibrate;
 
-  lpr_hid.graphics            = &lpr_graphics;
-
   hid_register_hid (&lpr_hid);
 }
